random/locked.go

```diff
@@ -0,0 +1,35 @@
+package random
+
+import (
+	"math/rand"
+	"sync"
+)
+
+// NewLocked returns a rand.Rand with a random source protected by a mutex. To be used for concurrent access.
+func NewLocked() *rand.Rand {
+	return rand.New(NewLockedSource(NewSource()))
+}
+
+// NewLockedSource wraps the with a mutex. To be used for concurrent access.
```

Comment:
Are you missing a word here maybe? 👍 
`NewLockedSource wraps the ____ with a mutex`?